db-async-common/src/main/java/com/github/jasync/sql/db/ConnectionPoolConfiguration.kt

```diff
@@ -56,6 +57,7 @@ data class ConnectionPoolConfiguration @JvmOverloads constructor(
     val username: String = "dbuser",
     val password: String? = null,
     val maxActiveConnections: Int = 1,
+    val maxTtl: Long = -1,
```

Comment:
I prefer calling it maxConnectionTtl. In addition, it is better to use nullable type with null as default to represent "no value". Please also add validation on allowed values (see below)

Reply:
I'll call it `maxObjectTtl` in `PoolConfiguration` and `maxConnectionTtl` in the ConnectionPoolConfiguration/Builder, ok?

Reply:
👍